gemrb/core/Scriptable/Actor.cpp

```diff
@@ -878,12 +878,14 @@ static void pcf_morale (Actor *actor, ieDword /*oldValue*/, ieDword /*newValue*/
 		// recover from panic, since morale has risen again
 		// but only if we have really just recovered, so panic from other
 		// sources isn't affected
-		if ((actor->Modified[IE_MORALE]-1 == actor->Modified[IE_MORALEBREAK]) || (actor->Modified[IE_MORALEBREAK] == 0) ) {
+		if ((actor->Modified[IE_MORALE]-1 >= actor->Modified[IE_MORALEBREAK]) || (actor->Modified[IE_MORALEBREAK] == 0) ) {
```

Comment:
I'm not so sure this is correct — won't it break non-morale induced panic? Effect based one should be fine if they don't have permanent timing, since the stat will get updated each tick, but we also permanently set it for turn undead and Panic actions, potentially only once.

Reply:
Hm, then we'd maybe need a panic cause marker or something like this to address this? I'm not sure, the whole thing isn't trivial.

Reply:
IE_MORALERECOVERYTIME is not affected by effects, so we could zero it for the other cases, so normal passage of time won't affect morale. And add a check here to skip such cases, since there are other sources of morale.

Reply:
Ok, I do the following since there is probably more than I can figure out this way or fix correctly: I drop the 2nd commit so that the CREImporter is fixed and create the new ticket describing the remaining problems. Maybe you can have a look at this then?

Reply:
Yeah, that's fine.